image_explorer/image_explorer.py

```diff
@@ -245,6 +249,23 @@ def _get_background(self, xmltree):
             'height': background.get('height')
         })
 
+    def _replace_static_from_url(self, url):
+        if not url:
+            return url
+        try:
+            from static_replace import replace_static_urls
+        except ImportError:
+            return url
+
+        url = '"{}"'.format(url)
+        lms_relative_url = replace_static_urls(url, course_id=self.course_id)
+        return lms_relative_url.strip('"')
+
+    def _make_url_absolute(self, url):
+        lms_base = settings.ENV_TOKENS.get('LMS_BASE')
+        scheme = 'https' if settings.HTTPS == 'on' else 'http'
+        return '{}://{}{}'.format(scheme, lms_base, url)
```

Comment:
Could you please use [`urlparse.urljoin`](https://docs.python.org/2/library/urlparse.html#urlparse.urljoin) here instead to join the `lms_base` and `url`?  This will handle the case where the `background[src] url` is actually already an absolute URL (e.g. a URL on an external website).